util/alertifier_providers.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+// DCSO FEVER
+// Copyright (c) 2020, DCSO GmbH
+
+import (
+	"fmt"
+
+	"github.com/DCSO/fever/types"
+)
+
+// AlertJSONProviderHTTPURL is an AlertJSONProvider for HTTP URL matches.
+type AlertJSONProviderHTTPURL struct{}
+
+// GetAlertJSON returns the "alert" subobject for an alert EVE event.
+func (a AlertJSONProviderHTTPURL) GetAlertJSON(inputEvent types.Entry,
+	prefix string, ioc string) ([]byte, error) {
+	v := fmt.Sprintf("%s | %s | %s", inputEvent.HTTPMethod, inputEvent.HTTPHost,
+		inputEvent.HTTPUrl)
+	return GenericGetAlertObjForIoc(inputEvent, prefix, v,
+		"%s Possibly bad HTTP URL: %s")
+}
+
+// AlertJSONProviderHTTPHost is an AlertJSONProvider for HTTP Host header
+// matches.
+type AlertJSONProviderHTTPHost struct{}
+
+// GetAlertJSON returns the "alert" subobject for an alert EVE event.
+func (a AlertJSONProviderHTTPHost) GetAlertJSON(inputEvent types.Entry,
+	prefix string, ioc string) ([]byte, error) {
+	return GenericGetAlertObjForIoc(inputEvent, prefix, ioc,
+		"%s Possibly bad HTTP host: %s")
+}
+
+// AlertJSONProviderDNSReq is an AlertJSONProvider for DNS request matches.
+type AlertJSONProviderDNSReq struct{}
+
+// GetAlertJSON returns the "alert" subobject for an alert EVE event.
+func (a AlertJSONProviderDNSReq) GetAlertJSON(inputEvent types.Entry,
+	prefix string, ioc string) ([]byte, error) {
+	return GenericGetAlertObjForIoc(inputEvent, prefix, ioc,
+		"%s Possibly bad DNS lookup to %s")
+}
+
+// AlertJSONProviderDNSResp is an AlertJSONProvider for DNS response matches.
+type AlertJSONProviderDNSResp struct{}
+
+// GetAlertJSON returns the "alert" subobject for an alert EVE event.
+func (a AlertJSONProviderDNSResp) GetAlertJSON(inputEvent types.Entry,
+	prefix string, ioc string) ([]byte, error) {
+	return GenericGetAlertObjForIoc(inputEvent, prefix, ioc,
+		"%s Possibly bad DNS response for %s")
+}
+
+// AlertJSONProviderTLSSni is an AlertJSONProvider for TLS SNI matches.
+type AlertJSONProviderTLSSni struct{}
```

Comment:
```suggestion
type AlertJSONProviderTLSSNI struct{}
```

Just an idea, because SNI is typically used as an acronym that's not different from URL.

Reply:
.. on the other hand: other parts of the code use `TLSSni`.

Reply:
Done.